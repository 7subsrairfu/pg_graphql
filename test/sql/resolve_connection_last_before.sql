select jsonb_pretty(
    gql.dispatch($$
        {
          allAccounts(last: 2, before: "WyJhY2NvdW50IiwgM10=") {
            edges {
              node {
                nodeId
                id
              }
            }
          }
        }
    $$)
);
