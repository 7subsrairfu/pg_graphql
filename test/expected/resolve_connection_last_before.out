select jsonb_pretty(
    gql.dispatch($$
        {
          allAccounts(last: 2, before: "WyJhY2NvdW50IiwgM10=") {
            edges {
              node {
                nodeId
                id
              }
            }
          }
        }
    $$)
);
                       jsonb_pretty                       
----------------------------------------------------------
 {                                                       +
     "data": {                                           +
         "allAccounts": {                                +
             "edges": [                                  +
                 {                                       +
                     "node": {                           +
                         "id": 1,                        +
                         "nodeId": "WyJhY2NvdW50IiwgMV0="+
                     }                                   +
                 },                                      +
                 {                                       +
                     "node": {                           +
                         "id": 2,                        +
                         "nodeId": "WyJhY2NvdW50IiwgMl0="+
                     }                                   +
                 }                                       +
             ]                                           +
         }                                               +
     },                                                  +
     "errors": [                                         +
     ]                                                   +
 }
(1 row)

